src/Compilers/CSharp/Portable/Symbols/PropertySymbol.cs

```diff
@@ -218,6 +218,8 @@ public bool IsWriteOnly
 
         internal abstract bool MustCallMethodsDirectly { get; }
 
+        internal abstract bool HasUnscopedRefAttribute { get; }
```

Comment:
>HasUnscopedRefAttribute [](http://example.com/codeflow?start=31&length=23)

Is `PropertySymbol.HasUnscopedRefAttribute` necessary? Should the attribute be supported on properties in general, just on properties from source, or only on accessors directly?